CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
```

Comment:
If you are the only developer this approach is probably fine but if there are going to be other people working on it consider using towncrier as is done in middleware.

Reply:
I thought about it but is it not mainly Python-oriented?

Reply:
It's nothing to do with python other than you can configure it in a `pyproject.toml` file. You run the `towncrier` command to combine the fragments into a single file.

Reply:
Hmm, ok. I'll consider it for a future release. Are there instructions/best practices for the middleware approach beyond the notes on [DM-30291](https://rubinobs.atlassian.net/browse/DM-30291) and the [doc/changes dir](https://github.com/lsst/daf_butler/tree/main/doc/changes)? I think I've only gone through the process of add a new item once.